### PR TITLE
feat: Added Landing Zone example for an HTTPS Load Balancer.

### PR DESCRIPTION
**Pull request by borkodjurkovic-ssc**

### CODE REVIEW

**Review by davelanglois-ssc — APPROVED** [review]

File: examples/landing-zone-v2/configconnector/tier3/https-external-load-balancer/backend-service.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  resourceID: workload-name-backend-service # kpt-set: ${workload-name}-backend-service
+  # set a mig created from tier4 as backend
+  backend:
+    - balancingMode: RATE
+      capacityScaler: 0.9
+      maxRate: 10000
+      group:
+        instanceGroupRef:
+          # when using a MIG, the external ref must be used and is pointing to its 'instanceGroup' value
+          external: https://www.googleapis.com/compute/v1/projects/project-id/zones/northamerica-northeast1-a/instanceGroups/workload-name-instance-group-manager # kpt-set: https://www.googleapis.com/compute/beta/projects/${project-id}/zones/northamerica-northeast1-a/instanceGroups/${workload-name}-instance-group-manager
```
Comment (fmichaelobrien) [inline examples/landing-zone-v2/configconnector/tier3/https-external-load-balancer/backend-service.yaml:33]: minor zone name in kpt-set ${zone}

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

File: examples/landing-zone-v2/configconnector/tier3/https-external-load-balancer/backend-service.yaml (hunk 1 above)
Comment (borkodjurkovic-ssc) [reply examples/landing-zone-v2/configconnector/tier3/https-external-load-balancer/backend-service.yaml:33]: I don't see it in the setters.yaml file - or am I looking at the wrong place?

Comment (obriensystems) [reply examples/landing-zone-v2/configconnector/tier3/https-external-load-balancer/backend-service.yaml:33]: it's not defined in the setters.yaml - would need to be there before we could use it in line 33 - it is alright we can leave it at northamerica-northeast-1a

**Pull request merged**